jmbase/jmbase/support.py

```diff
@@ -222,8 +222,11 @@ def lookup_appdata_folder(appname):
         data_folder = path.expanduser(path.join("~", "." + appname + "/"))
     return data_folder
 
+def get_jm_version_str():
+    return "JoinMarket " + JM_CORE_VERSION
+
 def print_jm_version(option, opt_str, value, parser):
-    print("JoinMarket " + JM_CORE_VERSION)
+    print(get_jm_version_str())
```

Comment:
What's happening here with the arguments? There are 4 arguments to the function but they're all unused. (Btw I'm not sure it's worth breaking out the second function, it doesn't appear to offer an advantage? but the arguments question is the one I'm more curious about).

Reply:
That's a requirement for optparse callback. https://docs.python.org/3/library/optparse.html#optparse-option-callbacks

Reply:
Ah; makes sense, thanks.